packages/discord.js/src/structures/Guild.js

```diff
@@ -1138,6 +1138,27 @@ class Guild extends AnonymousGuild {
     return this;
   }
 
+  /**
+   * @typedef {Object} GuildMFALevelOptions
+   * @property {GuildMFALevel} level The MFA level
+   * @property {string} [reason] Reason for changing the guild's MFA level
+   */
+
+  /**
+   * Sets the guild's MFA level
+   * @param {GuildMFALevelOptions} options The MFA level
+   * @returns {Promise<Guild>}
+   */
+  async setMFALevel({ level, reason }) {
```

Comment:
```suggestion
  async setMFALevel(level, reason) {
```

Also update the JSDoc and remove the `GuildMFALevel` typedef above